src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,144 @@
+package duke;
+
+import duke.task.Task;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Stores all User Interface related methods and objects.
+ */
+public class Ui {
+
+    public static String divider = "______________________________________________________________________________";
+    public static String logo = "             *\n"
+            + "      o  o  / \\  o  o\n"
+            + "      |\\/ \\/   \\/ \\/|\n"
+            + "      |             |\n"
+            + "      |ooooooooooooo|\n"
+            + " __  _  ____  ____    ____      ____    ___   ____\n"
+            + "|  |/ ||    ||    \\  /    |    |    \\  /   \\ |    \\\n"
+            + "|  ' /  |  | |  _  ||   __|    |  o  )|     ||  o  )\n"
+            + "|    \\  |  | |  |  ||  |  |    |     ||  O  ||     |\n"
+            + "|     | |  | |  |  ||  |_ |    |  O  ||     ||  O  |\n"
+            + "|  .  | |  | |  |  ||     |    |     ||     ||     |\n"
+            + "|__|\\_||____||__|__||___,_|    |_____| \\___/ |_____|\n";
```

Comment:
Assuming these are constants (since they won't change), I think the variable names must be all uppercase (e. g. DIVIDER and LOGO)